src/validation/index.js

```diff
@@ -45,4 +45,23 @@ export default {
 
     return next();
   },
+
+  rememberMe: (req, res, next) => {
+
+    const errors = [];
+    const { rememberMe } = req.query;
+
+    if (rememberMe.toLowerCase() !== 'yes') {
+      errors.push(`user profile will not be auto retrieved next time. Choose "YES" to do so.`)
```

Comment:
Using backticks here leads to a linting as single quotes would be just enough

Reply:
Seriously??